client/src/App.tsx

```diff
@@ -46,6 +47,7 @@ function App(): JSX.Element {
                   <Route exact path="/listings" component={Listings} />
                   <Route exact path="/bookings" component={Bookings} />
                   <ProtectedRoute exact path="/user/:path" component={ProfileSettings} />
+                  <Route exact path="/requests" component={Request} />
```

Comment:
Please make this a protected route, although users who are not logged in won't be able to see any data but we wouldn't want them navigating to this page.